code/graphics/debug_sphere.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "globalincs/pstypes.h"
+struct debug_sphere {
+	vec3d pos;
+	float radius;
+};
+
+namespace debug_spheres {
+	void add(vec3d pos, float radius);
+	void clear();
+	void render();
+}
```

Comment:
This probably looks weird to you and it's not a big deal, but I think you can fix them by pressing enter after the ending brace, iirc.

Reply:
It's not showing up here, but I mean the red circle that appears if you go to the files changed tab.